protoc_gen_d/protoc-gen-d.d

```diff
@@ -426,15 +429,14 @@ class CodeGenerator
         import std.format : format;
 
         auto fieldTypeName = typeName(field);
-        if (fieldTypeName.endsWith("]"))
-            return " = protoDefaultValue!(%s)".format(fieldTypeName);
-        else
-            return " = protoDefaultValue!%s".format(fieldTypeName);
+        return " = protoDefaultValue!(%s)".format(fieldTypeName);
```

Comment:
Why adding the `()` if not necessary.

Reply:
Eventually add `()` if `fieldTypeName` contains a `.`.

Reply:
OK. Done.